doc/source/whatsnew/v0.19.0.txt

```diff
@@ -861,6 +861,7 @@ Bug Fixes
 - Bug in ``io.json.json_normalize()``, where non-ascii keys raised an exception (:issue:`13213`)
 - Bug when passing a not-default-indexed ``Series`` as ``xerr`` or ``yerr`` in ``.plot()`` (:issue:`11858`)
 - Bug in area plot draws legend incorrectly if subplot is enabled or legend is moved after plot (matplotlib 1.5.0 is required to draw area plot legend properly) (issue:`9161`, :issue:`13544`)
+- Bug in ``DataFrame``, assigning a string index as a column once can make further assignments affect both index and column (:issue:`13522`)
```

Comment:
maybe something like: Bug in ``DataFrame`` assignment with an object-dtyped ``Index`` where the resultant column is mutable to the original object.